src/tilemaps/components/HexagonalGetTileCorners.js

```diff
@@ -54,13 +54,15 @@ var HexagonalGetTileCorners = function (tileX, tileY, camera, layer)
     {
         hexWidth = tileWidth / 2;
         hexHeight = b0 * tileHeight;
+        rotation = 30 * Math.PI /180;
     }
 
     for (var i = 0; i < 6; i++)
     {
         var angle = 2 * Math.PI * (0.5 - i) / 6;
+        angle += rotation;
 
-        corners.push(new Vector2(center.x + (hexWidth * Math.cos(angle)), center.y + (hexHeight * Math.sin(angle))));
+        corners.push(new Phaser.Math.Vector2(center.x + (hexWidth * Math.cos(angle)), center.y + (hexHeight * Math.sin(angle))));
```

Comment:
Please don't do this. Require the Vector2 and use it as per the original code. You cannot guarantee Phaser will exist in the global scope.

Reply:
Good Point